migreringer/src/main/resources/db/migration/defaultDS/4.1/V4.1_31__TFP-5732-medlemskapsvurdering.sql

```diff
@@ -0,0 +1,27 @@
+CREATE TABLE MEDLEMSKAPSVILKÅR_VURDERING
```

Comment:
Skal vi holde oss til A-Z i tabellnavn og sekvens + kalle den VILKAR_MEDLEMSKAP slik at vi skiller den fra øvrige medlemstabeller.
Opptjening (2 tabeller) er kandidat til samles samme sted VILKAR_OPPTJENING. Ease-of-navigation